packages/medusa/src/services/cart.ts

```diff
@@ -994,7 +994,7 @@ class CartService extends BaseService {
    */
   async applyDiscount(cart: Cart, discountCode: string): Promise<void> {
     const discount = await this.discountService_.retrieveByCode(discountCode, [
-      "rule",
+      // "rule",
```

Comment:
assuming this is commented on purpose for a later PR?

Reply:
Yes, [here](https://github.com/medusajs/medusa/issues/1181).